### PR TITLE
Add test case to bring down network interface of FA Controller

### PR DESCRIPTION
**Pull request by vprabhakar-px**
bring down network interface and bring back

### CODE REVIEW

File: tests/basic/pure_test.go [hunk 1]
```diff
@@ -2879,3 +2879,157 @@ var _ = Describe("{FBDAMultiTenancyBasicTest}", func() {
...
+			"Verify Pod Recovers from RO mode after Bounce after blocking network interface from FA end",
+			nil, 0)
+	})
+
+	itLog := "FADAPodRecoveryDisableDataPortsOnFA"
+	It(itLog, func() {
+
+		var contexts []*scheduler.Context
+		var k8sCore = core.Instance()
+		//podNodes := []*corev1.Pod{}
```
Comment (lsrinivas-pure) [inline tests/basic/pure_test.go:2902]: delete if not required

File: tests/common.go [hunk 2]
```diff
@@ -12170,3 +12170,128 @@ func WaitForVolumeClean(vol *volume.Volume) error {
...
+		return nil, fmt.Errorf("Unable to get Px Pure Secret")
+	}
+
+	if len(pxPureSecret.Arrays) > 0 {
+		return pxPureSecret.Arrays, nil
+	}
+	return nil, fmt.Errorf("Failed to list FA Arrays ")
+}
+
+func FlashArrayGetIscsiPorts() (map[string][]string, error) {
```
Comment (lsrinivas-pure) [inline tests/common.go:12193]: can you move all these to pureutils as these are specifc to FA/FB

Comment (vprabhakar-px) [reply tests/common.go:12193]: sure , will move it to pure utils

File: tests/basic/pure_test.go (hunk 1 above)
Comment (vprabhakar-px) [reply tests/basic/pure_test.go:2902]: done

File: tests/common.go (hunk 2 above)
Comment (vprabhakar-px) [reply tests/common.go:12193]: @lsrinivas-pure , this is the common function that is derived from functions listed in pure utils , I guess this should be in common utils . 
will be using this functions on multiple scripts belongs to FA going forward

**Review by lsrinivas-pure — APPROVED** [review]

**Pull request merged**